app/mailers/application_mailer.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: "hello@#{ENV['EMAIL_DOMAIN']}"
+  default from: "alanw@codeforamerica.org"
```

Comment:
I am not sure this will work since `codeforamerica.org` is not setup on mailgun as a verified domain: http://mailgun-documentation.readthedocs.io/en/latest/user_manual.html#verifying-your-domain

But we could set his email address as the `reply_to`, I think?

Reply:
Oh even better. Fixed in bf58da2.